config/rbac/base/role.yaml

```diff
@@ -0,0 +1,212 @@
+
```

Comment:
What happened to `config/rbac/role.yaml`?

Reply:
see other comment

Reply:
See my comment. :smile: